tracing/tracing.go

```diff
@@ -59,7 +59,7 @@ import (
 )
 
 // LoadPlugin loads the given opentracing plugin and returns an opentracing.Tracer
-func LoadPlugin(pluginDir string, opts []string) (ot.Tracer, error) {
+func LoadPlugin(pluginDirs []string, opts []string) (ot.Tracer, error) {
```

Comment:
An internal function or exposed "LoadPlugins" could do the file traversal and call the exposed LoadPlugin(), such that you do not need a breaking change